### ISSUE TITLE
installing lscg with default settings may cause some serious problems

### ISSUE THREAD

**Issue by bloodstone1476** [littlesera/LSCG#331]
for some people like breathplay or custom activities, so they will pay attention to the settings of these functions, but wont pay attention to the default settings of other functions（like magic or Hypnosis）.
the functions of hypnotic and magic under **the default settings** seems could to be **maliciously exploited by others**,In the case i met, a friend of mine that has installed the default lscg settings but has been modified by others cannot use safewords and send private messages in any difficulty mode or regardless of whether it is willing or not. it took us a lot of effort to help her restore these settings.

**Comment by littlesera** [comment]
Related to #332

**Comment by bloodstone1476** [comment]
> Related to #332

thx

**Comment by littlesera** [comment]
Thanks for letting me know, I'm committed to investigating this issue, but curious if I can get details on what this malicious user did. What were your steps to recover your friend's settings? LSCG itself does not modify any vanilla BC settings like difficulty etc.

By default, LSCG's 'magic' and 'hypnosis' should be disabled and not allow remote access through the user's profile dialog.

**Comment by littlesera** [comment]
LSCG 'Remote Access' commands only update settings for the LSCG "Triggered Hypnosis," "Breathplay," and "Magic" sections. This access should be blocked by default in the UI, and even if someone were to spoof the lscg remote command it should only update LSCG settings and either uninstalling the mod, using the "reset" button in LSCG settings page, or using "/lscg emergency" command should free the character from any LSCG restrictions.

**Comment by bloodstone1476** [comment]
when “Miss Bondage [ChrissyMarie]31281” enter the room,my friend was tied up instantly and said she was unable to interact to any extent,including clicking anywhere on the screen, using safewords (on normal difficulty), and she could only type the words with 【*】. For this reason, she commissioned me to find Others restored her state through the magic of lscg.
i didn't record much, that's all I know, but the victim at that time is now undergoing testing and should get back to you in #332 in a while i suppose

**Comment by littlesera** [comment]
do they have BCX installed as well?

**Comment by bloodstone1476** [comment]
> do they have BCX installed as well?

yes

**Comment by littlesera** [comment]
When she tried to speak were there LSCG emotes firing? or did she see local messages informing her of her inability?
Any BCX rules/curses that are maybe set to activate when ChrissyMarie is in the same room?

LSCG doesn't have any code that prevents safewords, but BCX does which is my suspicion.

**Comment by bloodstone1476** [comment]
> LSCG 'Remote Access' commands only update settings for the LSCG "Triggered Hypnosis," "Breathplay," and "Magic" sections. This access should be blocked by default in the UI, and even if someone were to spoof the lscg remote command it should only update LSCG settings and either uninstalling the mod, using the "reset" button in LSCG settings page, or using "/lscg emergency" command should free the character from any LSCG restrictions.

ty for the reminding, but Rosalyn told me that they can modify my settings, and I have not changed anything since I installed lscg(here in github),im not sure how it happened

**Comment by littlesera** [comment]
are you in BC testing right now?

**Comment by bloodstone1476** [comment]
> are you in BC testing right now?

yes they are,but it looked strange, after a bunch of descriptions of the magic of lscg there was no response anymore. I asked them and they didn't reply to me.

**Comment by bloodstone1476** [comment]
> When she tried to speak were there LSCG emotes firing? or did she see local messages informing her of her inability? Any BCX rules/curses that are maybe set to activate when ChrissyMarie is in the same room?
> 
> LSCG doesn't have any code that prevents safewords, but BCX does which is my suspicion.

she said she checked afterward and found no data packet.

**Comment by bloodstone1476** [comment]
there is automatic reply of lscg ,she said,for the summarize will be given in #332 i suppose,she is summarizing now

**Comment by littlesera** [comment]
Great, closing this issue as a dupe

**Comment by bloodstone1476** [comment]
> Great, closing this issue as a dupe

thx
